index.bs

```diff
@@ -5642,7 +5649,8 @@ To <dfn>retrieve a value from an object store</dfn> with
 
 1. If |record| was not found, return undefined.
 
-1. Let |serialized| be of |record|'s [=/value=].
+1. Let |serialized| be |record|'s [=/value=]. If an error occurs while reading the value from the
```

Comment:
Can you update the prose introducing the algorithm to indicate that it may return an error (DOMException) ?

Reviewing the rest of the spec, https://w3c.github.io/IndexedDB/#asynchronously-execute-a-request does have prose like _"if result is an error ..."_ which should process this failure correctly. I note that if we ever allowed storing a DOMException then the behavior would be ambiguous and we'd need to be more explicit here (and elsewhere) about success vs. failure, but that's not currently a problem.

Reply:
> ... does have prose like "if result is an error ..." which should process this failure correctly.

Yes, I was banking on this prose to complete the picture of how the returned error gets surfaced out of `IDBRequest`, but I agree that explicitly indicating this in the retrieval algorithm would help. I'll update it.

Reply:
Done